### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by katklat**
Implemented one controller and adjusted the respective routes.
I implemented the picture upload already because the external image urls were loading to slowly or not at all

### CODE REVIEW

**Review by cheyder — APPROVED** [review]
Looks good to me! :)

File: app/Http/Controllers/SongController.php [hunk 1]
```diff
@@ -1,10 +1,125 @@
...
+
+        return redirect()->route('songs.index');
+    }
+    private function validateNewData()
+    {
+        return request()->validate([
+            'title' => 'required',
+            'artist' => 'required',
+            'entry' => 'min:3',
+            'emoji' => '',
```
Comment (tillschander) [inline app/Http/Controllers/SongController.php:113]: Du könntest diese Emoji-Felder als 'nullable' markieren https://laravel.com/docs/7.x/validation#a-note-on-optional-fields

File: resources/views/songs/create.blade.php [hunk 2]
```diff
@@ -1,22 +1,24 @@
 @extends('layouts/app')
 @section('content')
 
+<!-- issue: chosen file/emoji is not shown in input -->
+<!-- ToDo: add basket selection -->
```
Comment (tillschander) [inline resources/views/songs/create.blade.php:5]: Might wanna add these TODOs/BUGs to your Kanban-Board

File: routes/web.php [hunk 3]
```diff
@@ -38,19 +38,7 @@
...
-Route::get('/songs/show', function () {
-    return view('songs.show');
-})->name('songs.show');
-Route::get('/songs/edit', function () {
-    return view('songs.edit');
-})->name('songs.edit');
-Route::get('/songs/create', function () {
-    return view('songs.create');
-})->name('songs.create');
+Route::resource('/songs', 'SongController');
```
Comment (tillschander) [inline routes/web.php:41]: nice

**Review by tillschander — APPROVED** [review]

File: app/Http/Controllers/SongController.php (hunk 1 above)
Comment (katklat) [reply app/Http/Controllers/SongController.php:113]: will do so for the second controller and adjust this one

**Pull request closed without merging**